notebook/debugging/data_similarity.ipynb

```diff
@@ -0,0 +1,1306 @@
+       "      <td>-142.139726</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>...</th>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>41</th>\n",
+       "      <td>GSE45923</td>\n",
+       "      <td>336.665715</td>\n",
+       "      <td>244.603096</td>\n",
+       "      <td>92.062619</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>82</th>\n",
+       "      <td>SRP077975</td>\n",
+       "      <td>336.233798</td>\n",
+       "      <td>243.975919</td>\n",
+       "      <td>92.257879</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>15</th>\n",
+       "      <td>GSE21592</td>\n",
+       "      <td>329.633644</td>\n",
+       "      <td>230.418254</td>\n",
+       "      <td>99.215390</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>9</th>\n",
+       "      <td>GSE14844</td>\n",
+       "      <td>332.476449</td>\n",
+       "      <td>232.749692</td>\n",
+       "      <td>99.726757</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>47</th>\n",
+       "      <td>GSE55201</td>\n",
+       "      <td>322.071044</td>\n",
+       "      <td>202.855416</td>\n",
+       "      <td>119.215628</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>87 rows × 4 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   study_name  distance_to_other  distance_to_same  difference\n",
+       "38   GSE42834         877.009616       1196.615115 -319.605499\n",
+       "35   GSE40553         901.551269       1207.377101 -305.825832\n",
+       "69   GSE83456         891.123738       1190.250458 -299.126720\n",
+       "50   GSE58411         880.542332       1172.460873 -291.918541\n",
+       "80  SRP049820         602.853087        744.992813 -142.139726\n",
+       "..        ...                ...               ...         ...\n",
+       "41   GSE45923         336.665715        244.603096   92.062619\n",
+       "82  SRP077975         336.233798        243.975919   92.257879\n",
+       "15   GSE21592         329.633644        230.418254   99.215390\n",
+       "9    GSE14844         332.476449        232.749692   99.726757\n",
+       "47   GSE55201         322.071044        202.855416  119.215628\n",
+       "\n",
+       "[87 rows x 4 columns]"
+      ]
+     },
+     "execution_count": 34,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "means_df.sort_values(by='difference')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These results indicate that most of the data is behaving as expected (the distance between pairs of samples from different studies is less than the distance between pairs of samples within the same study).\n",
```

Comment:
I would expect the distance between samples within the same study to be greater than the distance of two samples from different studies based on [these results](https://github.com/greenelab/whistl/blob/master/notebook/data_exploration/compendium_eda.ipynb). Very positive outliers would also be interesting because they would indicate that a study's samples are very similar to each other